api/client/webclient/webclient.go

```diff
@@ -185,8 +185,14 @@ func Ping(cfg *Config) (*PingResponse, error) {
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-
 	defer resp.Body.Close()
+	if resp.StatusCode == 400 {
```

Comment:
There should be a constant in `package net/http` so you don't have to hard-code the 400.